lib/ice_cube/time_util.rb

```diff
@@ -1,5 +1,7 @@
 require "date"
 require "time"
+require "active_support"
+require "active_support/core_ext"
```

Comment:
It looks like upstream won't accept this code if it uses ActiveSupport, since they want to keep it pure ruby.

https://github.com/ice-cube-ruby/ice_cube/pull/449#discussion_r1067495750
https://github.com/ice-cube-ruby/ice_cube/pull/449#discussion_r1066391771
https://github.com/ice-cube-ruby/ice_cube/pull/449#discussion_r1067497148

So if we did want to push this upstream we might need to replace them with pure ruby, but since that PR is still getting updates maybe we won't need to. So we can keep ActiveSupport for now and wait for upstream to fix & merge their version.

Reply:
> Achieve functionality first then optimise later.

That's a good rule.  The only thing is that tech-debt can hide under the radar.  But, still, you probably want the functionality first.  So,

> Achieve functionality first then remove tech-debt and optimise later.

It would be nice to push this upstream, but we don't really need to.  So, let's deal with that later too.  👍